### DISCUSSION TITLE
Nächste Baustelle....

### DISCUSSION THREAD

**Discussion by schimmi70** [obermacker/spacefights#12] [General]
Wo haben wir denn jetzt noch eine große Baustelle ? 
Ich denke mal, dass die Seite Rohstoffe als nächstes gemacht werden sollte.
Flotte funktioniert  soweit, braucht (wenn du nichts dagegen hast) aber mal ein neues Outfit ;)

Ich würde ansonsten mal die alten Fehlerlisten bei Trello durchgehen und schauen was wir machen können.

**Comment by obermacker** [comment]
Einstellungen:

- Passwort ändern
- Account löschen
- vielleicht doch Platz für eine optionale E-Mail Adresse für public Server
- Namen der Kolonien ändern
- Bei der Initalisierung für eine Galaxy fehlt das Fehlerfeedback. Er lehnt manchmal Koordinaten & Usernamen ohne Fehlermeldung ab

Andere:
- Imperium
- Koordinatenbuch
- Responsive Layout
- ~~Firefox Developer Edition hat (bei mir) Probleme, Dein Layout für Flotten richtig dazustellen~~
- Runden Start & Ende (+Timer)
- Wartungsmodus
- der Wechsel zwischen den Galaxien

Für Leser, hier das erwähnte Trello Board [https://trello.com/b/g09kBNs5/spacefightsorg-public](https://trello.com/b/g09kBNs5/spacefightsorg-public)

**Reply by schimmi70** [reply]
Ich denke Responsive Layout sollten wir bei neuen Seiten immer im Blick auf das Responsive Layout haben, aber nur im Blick auf die Zukunft. Ist an sonsten aus meiner Sicht nicht vorangig. 
Wir sollten das Spiel spielbar machen und als erstes die noch nicht fertigen Routinen fertig stellen, die fehlenden ergänzen.
lass uns abstimmen wer sich als nächstes um welche Teile kümmert.

- Was willst Du machen ?
- Was kann ich machen ?

> * Firefox Developer Edition hat (bei mir) Probleme, Dein Layout für Flotten richtig dazustellen

meinst du den Raumschiffbau / welche Version von Firefox ist es denn genau... habe hier mit 122.0.1 (64-Bit) keine Probleme / sonst mach mal ein issue auf.... mit Details...

**Comment by obermacker** [comment]
Das mit dem Layout schien an meinem `laragon` server zu liegen. Da ich den auf usb immer mit mir rumschleppe hat sich das so angefühlt als wäre FF dran Schuld.

Der Wartungsmodus wäre nice, der fehlt mir öfter mal.

Was magst Du machen? Ich bin noch die Tage am Kampfsystem dran.

**Reply by schimmi70** [reply]
Was meinst du mit Wartungsmodus...

Wie du ja merkst bin ich zur Zeit auch nicht der fleißigste (bzw. in meiner Freizeit ausgelastet).
Wie gesagt ich würde erst mal für das Spiel wichtige Funktion vorziehen und dann nach die Funktionen die nur rudimentär (von der Benutzeroberfläche) da sind erneuern.

Fehlen und für das Spiel wichtig wären zunächst:

- Koordinatenbuch
- Einstellungen

danach:

- Allianzen
- Handel
- Kampfstatistik

Oberflächentechnisch sehe ich gerade den Bereich:

- Rohstoffe (Rohstoffbunker funktioniert noch nicht)
- Flotte  (Oberfläche)

Wie siehst das ?

**Comment by obermacker** [comment]
Die Reinfolge ist super. Beim Koordinatenbuch hatte ich mir mal einen Teilenbutton, welcher eine Nachricht an einen anderen Spieler vorbereitet überlegt. Und das später die Koords für die Allianz geteilt werden können.

**Reply by schimmi70** [reply]
Ich kann mich nicht wirklich an das/ein Koordinatenbuch erinnern.
Kannst du mir grob sagen was rein soll, bzw. angezeigt werden soll. 
OK, Weitergabe der Koordinaten ist klar (mit Auswahl was weiter geben werden soll)
Soll dann gleich das Koordbuch vom entsprechenden Spieler ergänzt werden ?
Soll die Daten aus dem Koordbuch dann auch in der Sternenkarte angezeigt werden ?

VG Alban

**Comment by obermacker** [comment]
Guten Morgen, 

ich habe leider keine Bilder mehr davon. 

Das war damals nur ein simples Adressbuch, in dem man manuell die Koordinaten und den Namen eintragen konnte. Fest nach Namen sortiert wurde es eingeführt damit man nicht immer Koordinaten irgendwo außerhalb des Spiels dokumentieren oder den Spieler innerhalb der 14 Tage Erkundungs-Verfallszeit spionieren/angreifen musste.

> Soll dann gleich das Koordbuch vom entsprechenden Spieler ergänzt werden ?

Ich denke es sollte im Koordbuch nichts automatisch passieren. Keine Aktualisierung wenn der Spieler sich z.B. umbenennt oder löscht. Keine automatischen Vervollständigungen.

> Soll die Daten aus dem Koordbuch dann auch in der Sternenkarte angezeigt werden ?

Auch das würde ich vermeiden, sonst bräuchte man die Verfallszeit nicht und könnte die Gala immer aufgedeckt lassen.

Im Grunde soll das Buch nur ein kleines manuelles Hilfsmittel sein um sich die Adressen nicht merken oder anderweitig notieren zu müssen. 

Toll wäre:
-	Anklickbare Koordinaten die ins System oder in das Flottenmenü führen
-	Teilen der Koordinaten mit anderen Spielern
-	Kleines Bemerkungsfeld für eigene Notizen (wenige Zeichen)

VG Steve
